src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -521,6 +521,51 @@ public function cleanup(): void
         }
     }
 
+    public function getMessageCount(): int
+    {
+        $groups = $this->connection->xinfo('GROUPS', $this->stream);
+
+        if (false === $groups) {
+            throw new LogicException(sprintf('No group found for stream "%s".', $this->stream));
+        }
+
+        $lastDeliveredId = null;
+        foreach ($groups as $group) {
+            if ($group['name'] !== $this->group) {
+                continue;
+            }
+
+            // Use "lag" key provided by Redis 7.x. See https://redis.io/commands/xinfo-groups/#consumer-group-lag.
+            if (isset($group['lag'])) {
+                return $group['lag'];
+            }
+
+            $lastDeliveredId = $group['last-delivered-id'];
```

Comment:
Older php redis extension did not provide this information. See example: https://github.com/phpredis/phpredis/issues/1559

So we would maybe check if this is set and else also return a LogicException. Then the check for null === $lastDeliveredId should be obselete as I think redis always return a value here even if it is `0-0` for all versions supported this format.

Reply:
Looks for me solved in Line 539 a isset was added:

https://github.com/symfony/symfony/pull/46229/files#diff-a722c1aa345575df9562fc76db87e4dc9d1e610f9084a26627b6289655e891d3R539

Reply:
Sorr, read too fast, thank you.